docs/release/release_0_2_9.rst

```diff
@@ -0,0 +1,59 @@
+Announcement: napari 0.2.9
+==========================
+
+We're happy to announce the release of napari 0.2.9!
+napari is a fast, interactive, multi-dimensional image viewer for Python.
+It's designed for browsing, annotating, and analyzing large multi-dimensional
+images. It's built on top of Qt (for the GUI), vispy (for performant GPU-based
+rendering), and the scientific Python stack (numpy, scipy).
+
+
+For more information, examples, and documentation, please visit our website:
+https://github.com/napari/napari
+
+Highlights
+**********
+- better support for surface timeseries (#831)
+- contrast limits slider popup on right click (#837)
+- better isosurface rendering with colormaps (#840)
+- attenuated MIP mode for better 3D rendering (#846)
+
+New Features
+************
+- convert layer properties to dictionary (#686)
+- better support for surface timeseries (#831)
+- make contrast_limits_range public and climSlider popup on right click (#837)
+- attenuated MIP mode for better 3D rendering (#846)
+
+Improvements
+************
+- bump numpydoc dependency to 0_9_2 for faster startup (#830)
+- better isosurface rendering with colormaps (#840)
+- add nearest interpolation mode to volume rendering for better labels support (#841)
+- refactor RangeSlider to accept data range and values. (#844)
+- in bindings logic, check if generator, not generator function (#853)
+
+Bugfixes
+********
+- fix fullscreen crash for test_viewer (#849)
+- fix RangeSlider.rangeChange emit type bug (#856)
+
+API Changes
+***********
+- rename * to current_* and *s to * for certain layer properties in particular `edge_color` and `face_color`(#686)
```

Comment:
what does this mean? I'm confused

also it renders strangely with a link, which you can see [here](https://github.com/napari/napari/blob/b12639589143e9eeaed4a9cc4fb94faa878037af/docs/release/release_0_2_9.rst)

Reply:
good catch, that wasn't so clear. I've tried making it more explicit